src/main/java/com/segment/analytics/android/integrations/amplitude/AmplitudeIntegration.java

```diff
@@ -73,19 +74,27 @@ public String key() {
   // abstraction to mock what AmplitudeClient.getInstance() returns.
   interface Provider {
 
-    AmplitudeClient get();
+    AmplitudeClient getAmplitudeClient();
+
+    Identify getIdentify();
```

Comment:
Do you not need to initialize Identify with `new Identify();` before `getIdentify`?

Reply:
Ah good question. This is an interface, so we first need to define the methods it contains. `getIdentify()` here doesn't do anything  - REAL is the implementation of the interface, where we fill in the methods - i.e. with `return new Identify()`.